modules/serverless-logic-openAPI-function-definition.adoc

```diff
@@ -28,7 +28,7 @@ The `operation` attribute is a `string` composed of the following parameters:
 
 {ServerlessLogicProductName} supports the following URI schemes:
 
-* classpath: Use this for files located in the `src/main/resources` folder of the application project. `classpath` is the default URI scheme. If you do not define a URI scheme, the file location is `src/main/resources/myopenapifile.yaml`.
+* classpath: Use this for files located in the `<project_application_dir>/specs` folder of the application project. `classpath` is the default URI scheme. If you do not define a URI scheme, the file location is `<project_application_dir>/specs/myopenapifile.yaml`.
```

Comment:
@domhanak one observation here.

I think we have two situations:

1) when we have a Quarkus project, then OpenAPI files, must be located under src/main/resources directory as base. Since, this directory is included in the classpath root.
So, for example, we can place an OpenAPI file, in the src/main/resources/specs/my_function.yaml, and then referr it in the workflow as classspath:specs/my_function.yaml  or simply specs/my_function.yaml

2) When we have a Kn workflow CLI generated project.
In this case, given the nature of the project, which is not a Quarkus one, we use the specs directory directly.
And in this case, we must place the my_function.yaml, in the <project_applicaton_dir>/specs directly.
And we refer it from the workflow in the same way.

So I assume this change, is oriented to align to the Kn workflow plugin generated projects , right?

If that is the case, I'd recommend to adjust also the example.

{
   "functions": [
      {
         "name": "myFunction1",
         "operation": "classpath:/myopenapi-file.yaml#myFunction1"  <------- adjust this to use the specs directory
      }
   ]
}

Reply:
@wmedvede sure, will fix. The projects in OSL intend to be non-quarkus so thats why this update.